lapis2-docs/src/components/FieldsTable.astro

```diff
@@ -14,15 +14,16 @@ const config = getConfig();
     </thead>
     <tbody>
         {
-            config.schema.metadata.map((m) => (
+            config.schema.metadata.map((metadata) => (
                 <tr>
-                    <td>{m.name}</td>
+                    <td>{metadata.name}</td>
                     <td>
-                        {m.type}
-                        {m.name === config.schema.primaryKey && <> (primary key)</>}
+                        {metadata.type}
+                        {metadata.name === config.schema.primaryKey && <> (primary key)</>}
                     </td>
                     <td>
-                        <i>TODO</i>
+                        {(metadata.type === 'insertion' || metadata.type === 'aaInsertion') &&
```

Comment:
Do we not have the field "nucleotideInsertion" and "aminoAcidInsertion"? This should then be here instead of "insertion" and "aaInsertion"?

Reply:
As discussed - those are the field names. Let's leave it as it is for now.